rust/worker/src/execution/operator.rs

```diff
@@ -51,7 +51,7 @@ where
 {
     async fn run(&self) {
         let output = self.operator.run(&self.input).await;
-        let res = self.reply_channel.send(output).await;
+        let res = self.reply_channel.send(output, None).await;
```

Comment:
do we want to propagate a context here?

Reply:
Also curious about this one. What information is missing if we do pass in the tracing context?

Reply:
My thought process here was that this sends a reply back to the original component that had asked to execute this task. And that component should already be executing inside a span (after all that is how it passed its context to the worker which is now sending a reply).

Maybe there are patterns other than this for which this needs a non `None` value but I can come back to those once I build more context of the system?